wallet/wallet.go

```diff
@@ -502,3 +506,24 @@ func (w *Wallet) AddTransaction(id tx.ID) error {
 func (w *Wallet) GetHistory(addr string) []HistoryInfo {
 	return w.store.History.getAddrHistory(addr)
 }
+
+func (w *Wallet) AllAmount(addr string, pt payload.Type) int64 {
```

Comment:
Two notes:
1- The function name can change to "TotalAmount"
2- Let's add this in another PR. I think we may update the logic in a way to reduce the wasted changes.

Reply:
OK.